en/intro-concepts.md

```diff
@@ -210,11 +213,73 @@ switch (using Link Aggregation and Control Protocol, or LACP), or independently
 
 ### VLAN
 
-A *VLAN interface* can be used to connect to a tagged VLAN, if the switch port
-the node is connected to is authorized to access it.
+A *VLAN interface* can be used to connect to a tagged VLAN providing the switch
+port the node is connected to is authorized to access it.
```

Comment:
This is subjective and a tiny point, but I find this sentence harder to understand than the original. To me it reads as if the VLAN interface is connected to *provide* the switch port the node is connected to, rather than stating a condition.